gui/gui.go

```diff
@@ -334,8 +335,8 @@ func (ui *GUI) Run(ctx context.Context) {
 	// Use a ticker to periodically update cached data and push updates through
 	// any established websockets
```

Comment:
Comment should be updated because this goroutine is now doing more than just running the ticker and updating websockets.